assets/js/components/AnalyticsInactiveCTA.js

```diff
@@ -24,58 +24,24 @@ import PropTypes from 'prop-types';
 /**
  * WordPress dependencies
  */
-import { useCallback } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
-import {
-	showErrorNotification,
-} from '../util';
-import Data from 'googlesitekit-data';
-import CTA from './legacy-notifications/cta';
-import GenericError from './legacy-notifications/generic-error';
-import { STORE_NAME as CORE_USER, PERMISSION_MANAGE_OPTIONS } from '../googlesitekit/datastore/user/constants';
-import { STORE_NAME as CORE_MODULES } from '../googlesitekit/modules/datastore/constants';
-
-const { useSelect, useDispatch } = Data;
-
+import ActivateModuleCTA from './ActivateModuleCTA';
 function AnalyticsInactiveCTA( props ) {
```

Comment:
Nit-pick, but let's keep an empty line between imports and the main code (like the function here).

Reply:
Updated @felixarntz